translate/cloud-client/translate_v3_batch_translate_text_with_model_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import uuid
+import translate_v3_batch_translate_text_with_model
+from google.cloud import storage
+
+PROJECT_ID = os.environ['GCLOUD_PROJECT']
+MODEL_ID = os.environ['AUTOML_TRANSLATION_MODEL_ID']
```

Comment:
Do we have this set in the kokoro environment?
If so, sweet.

Reply:
Im not sure, it only exists inside java-docs-samples-testing project If model can be public, i think it might be possible

Reply:
doesn't look like this environment variable has been used before. likely doesn't exist.